### PR TITLE
Add property inspection pages

### PR DESCRIPTION
**Pull request by CJWaszczuk**

### CODE REVIEW

File: steps/id-and-residency/carer.tsx [hunk 1]
```diff
@@ -30,9 +30,9 @@ const step: ProcessStepDefinition = {
   heading: "Carer",
   step: {
     slug: PageSlugs.Carer,
-    nextSlug: PageSlugs.Submit,
+    nextSlug: PageSlugs.AboutProperty,
     Submit: makeSubmit({
-      href: hrefForSlug(PageSlugs.Submit),
+      href: hrefForSlug(PageSlugs.AboutProperty),
```
Comment (erbridge) [inline steps/id-and-residency/carer.tsx]: This should go back to the sections page.

File: steps/property-inspection/about-property.tsx [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    slug: PageSlugs.AboutProperty,
+    nextSlug: PageSlugs.Rooms,
+    Submit: makeSubmit({
+      href: hrefForSlug(PageSlugs.Rooms),
+      value: "Save and continue"
+    }),
+    componentWrappers: [
+      ComponentWrapper.wrapStatic(
+        new StaticComponent({
+          key: "floor-number-headinng",
```
Comment (erbridge) [inline steps/property-inspection/about-property.tsx]:
```suggestion
          key: "floor-number-heading",
```

File: steps/property-inspection/damage.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        })
+      ),
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "damage-images",
+          Component: ImageInput,
+          props: {
+            label: "Take photos of damage",
+            name: "damage-images",
+            hintText: "You can take up to 5 different photos" as
```
Comment (erbridge) [inline steps/property-inspection/damage.tsx]: Is this limit ok? I can imagine them needing more if there's a lot of damage. Check with Gill?

File: steps/property-inspection/structural-changes.tsx [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "has-structural-changes",
+          Component: RadioButtons,
+          props: {
+            name: "has-structural-changes",
+            legend: (
+              <FieldsetLegend>
+                Have any structural changes been made within the property
+                different from when originally let?
```
Comment (erbridge) [inline steps/property-inspection/structural-changes.tsx]: This sentence doesn't make sense.

File: steps/property-inspection/structural-changes.tsx [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+        })
+      ),
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "structural-changes-images",
+          Component: ImageInput,
+          props: {
+            label: "Take photos of structural changes and any documents",
+            name: "structural-changes-images",
+            hintText: "You can take up to 5 different photos" as
```
Comment (erbridge) [inline steps/property-inspection/structural-changes.tsx]: Again, is 5 enough?

File: steps/property-inspection/damage.tsx (hunk 3 above)
Comment (CJWaszczuk) [reply steps/property-inspection/damage.tsx]: Spoke to Gill about this, she seemed quite fine with 3 for everything. Figured the UR session on multiple images would root out whether they needed to upload more or not.

File: steps/property-inspection/smoke-alarm.tsx [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+import { TextArea } from "../../components/TextArea";
+import DatabaseSchema from "../../storage/DatabaseSchema";
+import processRef from "../../storage/processRef";
+
+import PageSlugs, { hrefForSlug } from "../PageSlugs";
+import PageTitles from "../PageTitles";
+
+const step: ProcessStepDefinition = {
+  title: PageTitles.SmokeAlarm,
+  heading: "Is there is a hard wired smoke alarm in the property?",
```
Comment (erbridge) [inline steps/property-inspection/smoke-alarm.tsx]: Heading.

File: steps/property-inspection/storing-materials.tsx [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+import DatabaseSchema from "../../storage/DatabaseSchema";
+import processRef from "../../storage/processRef";
+
+import PageSlugs, { hrefForSlug } from "../PageSlugs";
+import PageTitles from "../PageTitles";
+
+const step: ProcessStepDefinition = {
+  title: PageTitles.StoringMaterials,
+  heading:
+    "Is the tenant storing materials in their home that can catch fire, other than those needed for normal household use?",
```
Comment (erbridge) [inline steps/property-inspection/storing-materials.tsx]: Heading.

File: steps/property-inspection/structural-changes.tsx [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+import DatabaseSchema from "../../storage/DatabaseSchema";
+import processRef from "../../storage/processRef";
+
+import PageSlugs, { hrefForSlug } from "../PageSlugs";
+import PageTitles from "../PageTitles";
+
+const step: ProcessStepDefinition = {
+  title: PageTitles.StructuralChanges,
+  heading:
+    "Have any structural changes been made within the property different from when originally let?  ",
```
Comment (erbridge) [inline steps/property-inspection/structural-changes.tsx]: Heading.

**Comment by erbridge** [comment]
Also, please update the logic in the sections page for sections status.

File: steps/property-inspection/structural-changes.tsx (hunk 4 above)
Comment (erbridge) [reply steps/property-inspection/structural-changes.tsx]: This still needs attention.

**Review by erbridge — APPROVED** [review]

**Pull request merged**